scripts/release_issue_status/main.py

```diff
@@ -148,19 +145,23 @@ def auto_reply(item, request_repo, rest_repo, sdk_repo, duplicated_issue, python
             raise
     else:
         try:
-            readme_link, output_folder = find_readme_and_output_folder(request_repo, rest_repo, item.issue_object.number)
+            readme_link, output_folder = get_readme_and_output_folder(request_repo, rest_repo, item.issue_object.number)
         except Exception as e:
-            print('Issue: {}  updates body failed'.format(item.issue_object.number))
-            item.bot_advice = 'failed to find Readme link, Please check !!'
+            print('Issue: {}  get pkname and output folder failed'.format(item.issue_object.number))
+            item.bot_advice = 'failed to find Readme link and output folder. Please check !!'
             item.labels.append('attention')
             item.issue_object.set_labels(*item.labels)
+            print(e)
             raise
     try:
         print("*********************")
         print(python_piplines)
         pipeline_url = get_pipeline_url(python_piplines, output_folder)
         rg.begin_reply_generate(item=item, rest_repo=rest_repo, readme_link=readme_link,
                                 sdk_repo=sdk_repo, pipeline_url=pipeline_url)
+        if 'Configured' in item.labels:
+            item.labels.remove('Configured')
+            item.issue_object.set_labels(*item.labels)
```

Comment:
These seems be many place which need remove label. So think about extract one single function.